packages/cosmjs/src/cosmoskit/assetlist.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * A pre-configured asset list with the native Sei token for use with Cosmos Kit.
+ * @category Cosmos Kit
+ */
+export const COSMOS_KIT_ASSET_LIST = {
```

Comment:
can we make assets here extensible so developers can add more assets if they want?